src/components/playlist-header/index.tsx

```diff
@@ -25,9 +25,11 @@ interface PlaylistHeaderProps {
 export const PlaylistHeader = withText(translates)(({onClose, title, amount, duration, pluginMode, ...otherProps}: PlaylistHeaderProps) => {
   return (
     <div className={[styles.playlistHeader, pluginMode === PluginPositions.VERTICAL ? styles.vertical : styles.horizontal].join(' ')}>
-      <div className={styles.playlistMetadata}>
-        <div className={styles.playlistTitle} title={title}>{title}</div>
-        <div className={styles.playlistDuration}>{`${amount},${duration}`}</div>
+      <div className={styles.playlistMetadata} role="dialog" aria-live="polite">
```

Comment:
@semarche-kaltura - do we think that the title or duration will change during runtime? If not - I don't think we need the aria-live here